lib/platforms/web.dart

```diff
@@ -59,10 +62,11 @@ void _setWebTitle(dynamic appName) {
   }
 }
 
-void _setPWAAppName(dynamic appName) {
+void _setPWAAppName(dynamic appName, dynamic shortAppName) {
   try {
     if (appName == null) return;
     if (appName is! String) throw _PackageRenameErrors.invalidAppName;
+    final actualShortAppName = shortAppName is String ? shortAppName : appName;
```

Comment:
The checks are failing because the exception of invalid short name isn't used here.

Should update the code as:

```dart
if (appName == null) return;
if (appName is! String) throw _PackageRenameErrors.invalidAppName;

if (shortAppName != null && shortAppName is! String) {
  throw _PackageRenameErrors. invalidShortAppName;
}
final actualShortAppName = shortAppName ?? appName;
```

Reply:
good point, I've updated the code!